pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -121,6 +121,7 @@ let
         pahole
         perl
         elfutils
+        hexdump
```

Comment:
I tried building `vmlinuz.efi` on aarch64. This is actually wrong here since it needs to be put in `nativeBuildInputs`.